Gateway/Http/Client/TransactionInitialize.php

```diff
@@ -73,6 +82,12 @@ public function placeRequest(TransferInterface $transferObject): array
             }
 
             $input = $this->roundOrderValues($transferObject->getBody());
+
+            // Monitor the domain
```

Comment:
Same as the Woocommerce MR on these comments. 
I think the code already gives this context so the comments are not adding anything and can be removed.